### ISSUE TITLE
It would be nice to have a damage idicator

### ISSUE THREAD

**Issue by OttoRocket-LVP** [LVPlayground/playground#776]
Suggestion as per Reni:
It would be nice to see how much damage you did to somebody (As addition to the hitsound).

Note:
I've seen other servers using this. Using text draw shown under in the screen with the damage dealt.

**Comment by FinishaLVP** [comment]
Agreed! This is really needed for better deathmatch experience. Indicator shows the information of the player receiving damage and the damage value, and vice versa.

Hopefully this filterscript is what we need https://forum.sa-mp.com/showthread.php?t=351329
